pkgs/build-support/trivial-builders/default.nix

```diff
@@ -815,6 +815,36 @@ rec {
         writeDirectReferencesToFile (writeText "string-file" string);
 
 
+  /**
+   * Write the references (i.e. the runtime dependencies in the Nix store) of the elements in `paths' to a file.
+   */
```

Comment:
Could you move this to the manual, `doc/build-helpers/trivial-build-helpers.chapter.md`?

What to do was unclear, so I've opened
- https://github.com/NixOS/nixpkgs/pull/294944

Reply:
I just checked and `referencesByCount` expects `rootPaths` to be string-like